CHANGELOG.MD

```diff
@@ -0,0 +1,7 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [1.0.2] - 2016-04-13
```

Comment:
Need to add a link so this comparison will show up

Take a look at the raw markdown in one of the other projects